web/crux-ui/src/components/nodes/use-node-details-state.tsx

```diff
@@ -88,7 +112,12 @@ const useNodeDetailsState = (options: NodeDetailsStateOptions): [NodeDetailsStat
         it.imageTag,
         utcDateToLocale(it.createdAt),
       ]),
+      containerShowAllFilter,
     ],
+    initialFilter: {
+      text: '',
+      showAll: node.type === 'k8s',
```

Comment:
Why added k8s type in the initial filter?

Reply:
k8s containers are not labeled yet, so we show all containers for kube nodes.